BE/FunctionalEntities/FunctionalEntities.rdf

```diff
@@ -46,66 +46,46 @@
 		<fibo-fnd-utl-av:hasMaturityLevel rdf:resource="&fibo-fnd-utl-av;Release"/>
 	</owl:Ontology>
 	
-	<owl:Class rdf:about="&fibo-be-fct-fct;Business">
-		<rdfs:subClassOf rdf:resource="&fibo-be-fct-fct;FunctionalBusinessEntity"/>
-		<rdfs:subClassOf>
-			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;isCharacterizedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-be-fct-fct;CommercialActivity"/>
-			</owl:Restriction>
-		</rdfs:subClassOf>
-		<rdfs:label>business</rdfs:label>
-		<skos:definition>An organization or economic system where goods and services are exchanged for one another or for money. Every business requires some form of investment and enough customers to whom its output can be sold on a consistent basis in order to make a profit. Businesses can be privately owned, not-for-profit or state-owned. An example of a corporate business is PepsiCo, while a mom-and-pop catering business is a private enterprise.</skos:definition>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/business.html</fibo-fnd-utl-av:definitionOrigin>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-be-fct-fct;Commerce">
-		<rdfs:subClassOf rdf:resource="&fibo-be-fct-fct;CommercialActivity"/>
-		<rdfs:label>commerce</rdfs:label>
-		<skos:definition>the commercial activity of buying and selling goods</skos:definition>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-be-fct-fct;CommercialActivity">
-		<rdfs:label>commercial activity</rdfs:label>
-		<skos:definition>the context of carrying out trade and other commercial, i.e., for-profit activities</skos:definition>
-		<fibo-fnd-utl-av:explanatoryNote>This is the context which distinguishes businesses or commercial organizations from organizations in general, the latter including government, trans-national and non profit organizations. Note that these distinctions are usually made with reference to these kinds of context and are not necessarily reflected in the structure of those organizations.</fibo-fnd-utl-av:explanatoryNote>
-	</owl:Class>
-	
 	<owl:Class rdf:about="&fibo-be-fct-fct;CooperativeSociety">
 		<rdfs:subClassOf rdf:resource="&fibo-be-fct-fct;FunctionalBusinessEntity"/>
 		<rdfs:label>cooperative society</rdfs:label>
-		<skos:definition>a commercial enterprise owned and managed by and for the benefit of customers or workers</skos:definition>
+		<skos:definition>organization owned by and operated for the benefit of those using its services</skos:definition>
```

Comment:
this is a consumer coop. A more usual type is a worker coop where the consumers (those using its services) have no ownership.